### PR TITLE
Add two additional columns to DATEV_ExportLine

### PR DESCRIPTION
**Pull request by metas-ts**
which can be selected for exporting to csv via DATEV_ExportFormat:
- c_tax_rate
- c_doctype_name

refs #13664

### CODE REVIEW

File: backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/dao/impl/QueryInsertExecutor.java [hunk 1]
```diff
@@ -61,6 +62,7 @@
 		Check.assumeNotNull(fromQuery, "fromQuery not null");
 		this.fromQuery = fromQuery;
 		this.fromModelClass = fromQuery.getModelClass();
+		this.fromTableName = InterfaceWrapperHelper.getTableName(fromModelClass);
```
Comment (metas-ts) [inline backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/dao/impl/QueryInsertExecutor.java:65]: i can't comment on lines 54-58 but - boyscout-rule - please 
- annotate the two params as   `@NonNull`
- remove the Check.assumeNotNulls
- remove the super()

File: backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/model/InterfaceWrapperHelper.java [hunk 2]
```diff
@@ -1646,7 +1646,7 @@ public static Evaluatee getEvaluatee(final Object model)
 
 	/**
 	 * @param modelClass
-	 * @return immutable list of column names of modelClass's table
+	 * @return immutable list of physical column names of modelClass's table
 	 */
 	public static Set<String> getModelColumnNames(final Class<?> modelClass)
```
Comment (metas-ts) [inline backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/model/InterfaceWrapperHelper.java:1651]: please rename to `getModelPhysicalColumnNames()`

**Review by metas-ts — COMMENTED** [review]
(can't approve or request changes since i created the PR)
please chck out my AFAIU simple requests. Afterwards, feel free to intergrate without further ado

File: backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/dao/impl/QueryInsertExecutor.java (hunk 1 above)
Comment (adi-stefan) [reply backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/ad/dao/impl/QueryInsertExecutor.java:63]: done

File: backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/model/InterfaceWrapperHelper.java (hunk 2 above)
Comment (adi-stefan) [reply backend/de.metas.adempiere.adempiere/base/src/main/java/org/adempiere/model/InterfaceWrapperHelper.java]: done

**Pull request merged**